docs/source/platform/schema-registry.md

```diff
@@ -48,13 +46,13 @@ apollo service:push --key="<API_KEY>" --endpoint="https://example.com/graphql"
 
 <h3 id="viewing-schema">Viewing a published schema</h3>
 
-Now that the service has been pushed, it can be viewed by going to [Engine](https://engine.apollographql.com) and browsing to the service's dashboard. The schema that was pushed should now appear with overal information about its number of types and fields, as well as full information about every type, argument, and description of the schema. With this done, teams can now use productivity boosters like the [Apollo VS Code extension](./editor-plugins.html)
+Now that the service is pushed, view it on the [Apollo UI](https://engine.apollographql.com) by browsing to the service's dashboard. The pushed service should now appear with an overall schema summary about its types and fields, as well as full information about every type, argument, and description in the schema. With a published schema, teams can now use productivity boosters such as the [Apollo VS Code extension](./editor-plugins.html)
 
-<h2 id="schema-tags">Schema tags</h2>
+<h2 id="schema-tags">Coordinating with schema tags</h2>
 
-Product cycles move incredibly fast and coordination of teams is critical to shipping features quickly. To enable this coordination, the Apollo schema registry allows teams to push proposed or future versions of their schema to the registry so teams can use them in their editors, validate against them, and have a center point of truth even for the future of their graph.
+Product cycles move incredibly fast and coordination of teams is critical to shipping features quickly. To enable communication, the Apollo schema registry allows teams to push proposed or future versions of their schema to the registry. These new versions are used in editors, validation, and documentation as the source of truth for future schemas.
```

Comment:
This section doesn't explain what a tag actually is, but instead gives an example of what it's useful for?

Reply:
I think this section needs to be a bit more fleshed out (now that we're here; I recognize that this proposal is just a good copy change, but since this update ties so closely into our better understanding of schema tags, I think we should update this sections as part of it)